compiler/test/dotc/run-test-pickling.blacklist

```diff
@@ -1,6 +1,7 @@
 derive-generic.scala
 eff-dependent.scala
 enum-java
+i4192
```

Comment:
I'm not quite sure what are the criteria for skipping pickling tests - how can I know if this is OK or the failing pickling test is a sign that there was a bug somewhere else?

Reply:
It's always some sort of bug, but sometimes it's a bug in the way we run the test itself (when there's a "pickling difference" it means there's a difference in the pretty-printed trees, but sometimes that just means we need to adjust how we pretty-print these trees), it's a good idea to add a comment mentioning what kind of error lead to blacklisting the test in any case.

Reply:
I've found the reason of the pickling tests failure https://github.com/lampepfl/dotty/issues/11648 and applied a workaround not to disable the test